packages/devextreme/js/__internal/ui/m_tag_box.ts

```diff
@@ -1295,17 +1295,18 @@ const TagBox = (SelectBox as any).inherit({
       return;
     }
 
+    const isListRendered = !this.option('deferRendering');
```

Comment:
It seems that judging by the deferRendering option is not reliable that list was rendered. What will happen if the select box was already opened and closed? Will this code work?

Reply:
ok i will try to find alternative to check if list is rendered

Reply:
[30d85eb](https://github.com/DevExpress/DevExtreme/pull/27341/commits/30d85eb8b885fb8cfad6063c4b073a2970efa847)